python_builtin_functions.py

```diff
@@ -4,86 +4,170 @@ def myabs(number):
     return number
 
+
 def myall(iterable):
     """ Implementation of built-in `all` function """
-    pass
+    for item in iterable:
+        if not item:
+            return False
+    return True
 
 
 def myany(iterable):
     """ Implementation of built-in `any` function """
-    pass
+    for item in iterable:
+        if item:
+            return True
+    return False
 
 
 def mylen(s):
     """ Implementation of built-in `len` function """
-    pass
+    i=0
+    for _ in s:
+        i+=1
+    return i
 
 
 def mysum(iterable):
     """ Implementation of built-in `sum` function """
-    pass
+    i=0
+    for x in iterable:
+        i+=x
+    return i
 
 
 def mymax(iterable):
     """ Implementation of built-in `max` function """
-    pass
+    i=0
+    for x in iterable:
+        if x>i:
+            i=x
+        else:
+            continue
+    return i
 
 
 def mymin(iterable):
     """ Implementation of built-in `min` function """
-    pass
+    i=1000**1000
+    for x in iterable:
+        if x<i:
+            i=x
+        else:
+            continue
+    return i
 
 
 def mydivmod(a, b):
     """ Implementation of built-in `divmod` function """
-    pass
+    q=a//b
+    r=a%b
+    return (q,r)
 
 
 def mybool(x):
     """ Implementation of built-in `bool` function """
-    pass
+    if x:
+        return True
+    return False
 
 
 def mypow(base, exp):
     """ Implementation of built-in `pow` function """
-    pass
+    i=base**exp
+    return i
 
 
 def myreversed(seq):
     """ Implementation of built-in `reversed` function """
-    pass
+    str1=''
+    for i in range (1,len(seq)+1):
+        str1+=seq[-i]
+    return str1
 
 
 def myenumerate(iterable, start=0):
     """ Implementation of built-in `enumerate` function """
-    pass
+    lst=[]
+    tup=()
+    for x in range (0,len(iterable)):
+        tup=(x,iterable[x])
+        lst.append(tup)
+    return lst
 
 
 def mybin(x):
     """ Implementation of built-in `bin` function """
-    pass
+    lst=['0b']
+    while x>0:
+        r=x%2
+        x=x//2
+        lst.insert(1,str(r))
+    return ''.join(lst)
 
 
 def myhex(x):
     """ Implementation of built-in `hex` function """
-    pass
+    lst=['0x']
+    while int(x>0):
+        r=x%16
+        x=x//16
+        if r==10:
+            r='a'
+        elif r==11:
+            r='b'
+        elif r==12:
+            r='c'
+        elif r==13:
+            r='d'
+        elif r==14:
+            r='e'
+        elif r==15:
+            r='f'
+        else:
+            pass
+        lst.insert(1,str(r))
+    return ''.join(lst)
 
 
 def myoct(x):
     """ Implementation of built-in `oct` function """
-    pass
+    lst=['0o']
+    while int(x>0):
+        r=x%8
+        x=x//8
+        lst.insert(1,str(r))
+    return ''.join(lst)
 
 
 def myisinstance(object, classinfo):
     """ Implementation of built-in `isinstance` function """
-    pass
+    t1=type(object)
+    t2=str(t1).find(classinfo)
+    if t2>=0:
```

Comment:
Though this works, this is a bit confusing..
Try as below..

```python
if str(classinfo) in type(object):
    return True
return False
```

Reply:
Changed as per your instruction.